examples/fluids/src/setupdm.c

```diff
@@ -100,6 +100,14 @@ PetscErrorCode SetUpDM(DM dm, ProblemData *problem, PetscInt degree, SimpleBC bc
       PetscCall(PetscSectionSetComponentName(section, 0, 3, "Velocity Z"));
       PetscCall(PetscSectionSetComponentName(section, 0, 4, "Temperature"));
       break;
+
+    case STATEVAR_ENTROPY:
+      PetscCall(PetscSectionSetComponentName(section, 0, 0, "Chemical Potential"));
+      PetscCall(PetscSectionSetComponentName(section, 0, 1, "Velocity X"));
+      PetscCall(PetscSectionSetComponentName(section, 0, 2, "Velocity Y"));
+      PetscCall(PetscSectionSetComponentName(section, 0, 3, "Velocity Z"));
```

Comment:
I definitely would not call these `Velocity` outright, as they're not just velocity. This breaks CGNS naming convention (see table at bottom of section A.2 in [CGNS standard](https://cgns.github.io/CGNS_docs_current/sids/dataname.html#dataname_flow)).

The spaces also technically break the standard, but you're just following what was done above which is fine. Something to change at somepoint later.

Reply:
Oh wow, I've never seen that before. I was/am struggling to figure out meaningful names for these quantities.

Reply:
Yeah. I think being blunt with it isn't a bad option:
```
EntropyEnergy
EntropyVelocityX
EntropyVelocityY
EntropyVelocityZ
EntropyTemperature
```

Reply:
So these are gradients of entropy with respect to conservative variables and thus `EntropyEnergy` and `EntropyTemperature` feel confusing even though the terms left over with an ideal gas have that structure. I think the velocity-like terms should still be organized as vectors (with `X`, `Y`, `Z` suffixes), but names could even be
```
EntropyDensity
EntropyMomentumX
EntropyMomentumY
EntropyMomentumZ
EntropyEnergy
```
where the qualifier is the conservative variable that entropy is differentiated with respect to. In practice, we'll either need to use a calculator or convert to understandable variables to visualize, but that can be done in a new MR when it's more needed.

Reply:
Thanks Jed -- That makes more sense I think.